lib/connection.c

```diff
@@ -769,11 +796,24 @@ sf_bool STDCALL http_perform(CURL *curl,
 
     //TODO set error buffer
 
+    // Find request GUID in the supplied URL
+    char *request_guid_ptr = strstr(url, "request_guid=");
+    // Set pointer to the beginning of the UUID string if request GUID exists
+    if (request_guid_ptr) {
+        request_guid_ptr = request_guid_ptr + 13;
```

Comment:
can this 13 be a constant or add a comment here?

Reply:
Sure